main.js

```diff
@@ -140,10 +140,94 @@ function main() {
 	                    adapter.log.warn(error);
 	                }
 
-	                adapter.log.info("Done, stopping...");
-	                adapter.stop();

```

Comment:
does this work for you without stopping the adapter? It's scheduled, so it needs to be stopped after finishing